Communication/Packets/Incoming/Rooms/Settings/DeleteRoomEvent.cs

```diff
@@ -70,11 +70,8 @@ public void Parse(GameClient session, ClientPacket packet)
 
             using (IQueryAdapter dbClient = PlusEnvironment.GetDatabaseManager().GetQueryReactor())
             {
-                dbClient.RunQuery("DELETE FROM `user_roomvisits` WHERE `room_id` = '" + roomId + "'");
-                dbClient.RunQuery("DELETE FROM `rooms` WHERE `id` = '" + roomId + "' LIMIT 1");
-                dbClient.RunQuery("DELETE FROM `user_favorites` WHERE `room_id` = '" + roomId + "'");
+                dbClient.RunQuery("DELETE rooms,user_roomvisits,user_favorites,room_rights,room_bans,room_filter,room_visits,room_promotions FROM rooms LEFT JOIN user_roomvisits ON (rooms.id = user_roomvisits.room_id) LEFT JOIN user_favorites ON (rooms.id = user_favorites.room_id) LEFT JOIN room_rights ON (rooms.id = room_rights.room_id) LEFT JOIN room_bans ON (rooms.id = room_bans.room_id) LEFT JOIN room_filter ON (rooms.id = room_filter.room_id) LEFT JOIN room_visits ON (rooms.id = room_visits.room_id) LEFT JOIN room_promotions ON (rooms.id = room_promotions.room_id) WHERE rooms.id = " + room.Id);

```

Comment:
Lets not initiate in a debate, whether joins are plausible for Plus to work. They're not necessary, because the database is normalized. This is bad for performance.